content/support-training.html

```diff
@@ -0,0 +1,173 @@
+---
+title: Support & Training
+layout: jumbotron
+---
+
+<div class="container-fluid training">
+  
+  <div class="row gray-bg">
+    <div class="container">
+    <div class="col-md-12 doc-content">
+      <h1>Support & Training</h1>
+      <p>
+        Prometheus is an open source project. Training resources on this page are 
+        listed for your convenience only. These resources are independently 
+        created, and their content is not governed or verified by the
+        Prometheus project's maintainers.
+      </p>
+      
+      <p>If you'd like to add your training resource, see <a href="#add">Add a resource</a>.
+    </div>
+    </div>
+  </div>
+
+
+  <div class="row">
+    <div class="container">
+      <div class="col-md-12 doc-content training-team">
+        <h2>Training</h2>
+        <p>
+          These training resources are created and delivered independently 
+          by members of the Prometheus project team. Note that these do operate for a fee. This list is provided in alphabetical order.
```

Comment:
These are all independent of the project, even if some of the individuals involved happen to be on -team. We shouldn't be promoting any one course over any other.

I think we should only be providing plain links, as we presently do with the commercial support list.

Reply:
> These are all independent of the project, even if some of the individuals involved happen to be on -team. We shouldn't be promoting any one course over any other.

Yeah, I would just say that it's third-party trainings that are not officially endorsed by the Prometheus project.

> I think we should only be providing plain links, as we presently do with the commercial support list.

I don't mind this one either way, but I do like the company logo boxes in the Commercial Support section below at least.

Reply:
@brian-brazil @juliusv How about:

> These training resources are created and delivered by independent third parties. They operate for a fee. This list is provided in alphabetical order.

Reply:
Not all of them have a fee.

My concerns still stand - we shouldn't be promoting any one course over any other. In addition from a legal standpoint the Robust Perception training is not from me personally.

Reply:
Ah true, the first one in the RP series is a free training.

> My concerns still stand - we shouldn't be promoting any one course over any other.

I guess you mean getting rid of the statement that says that they're delivered by Prometheus team members? I think I agree... as long as people fulfill the requirements below for adding a course, we shouldn't be the judges here (too much potential for commercial conflict of interest).

Reply:
> I guess you mean getting rid of the statement that says that they're delivered by Prometheus team members?

Yes. If we're going alphabetical, then the only thing that should matter is the name of the relevant entity.

Reply:
Yeah, agreed.

Reply:
See [the latest commit](https://github.com/prometheus/docs/pull/1604/commits/156125db9bc70a82a1a368dc37215b917bcf5ce9) :)

Reply:
That's better, but the blurbs are still there. I don't want to have to police the wording of companies marketing content, as that's tricky from an impartiality standpoint. It'd also end up pretty bloated on the page. There should only be a list of organisation names.

Reply:
Ahh, I see. I misunderstood your comment. Check the latest commit.

Reply:
The headlines are still there, it should only be the company names. Otherwise RP could request for example an entry for each of our 9 courses.